modules/services/strongswan-swanctl/default.nix

```
@@ -0,0 +1,109 @@
+      mkdir -p '/etc/swanctl/pubkey'   # Raw public keys
+      mkdir -p '/etc/swanctl/private'  # Private keys in any format
+      mkdir -p '/etc/swanctl/rsa'      # PKCS#1 encoded RSA private keys
+      mkdir -p '/etc/swanctl/ecdsa'    # Plain ECDSA private keys
+      mkdir -p '/etc/swanctl/bliss'
+      mkdir -p '/etc/swanctl/pkcs8'    # PKCS#8 encoded private keys of any type
+      mkdir -p '/etc/swanctl/pkcs12'   # PKCS#12 containers
+
+      rm -rf ${pkgs.strongswan}/etc/swanctl/swanctl.conf
+      rm -rf ${pkgs.strongswan}/etc/swanctl/x509
+      rm -rf ${pkgs.strongswan}/etc/swanctl/x509ca
+      rm -rf ${pkgs.strongswan}/etc/swanctl/x509ocsp
+      rm -rf ${pkgs.strongswan}/etc/swanctl/x509aa
+      rm -rf ${pkgs.strongswan}/etc/swanctl/x509ac
+      rm -rf ${pkgs.strongswan}/etc/swanctl/x509crl
+      rm -rf ${pkgs.strongswan}/etc/swanctl/pubkey
+      rm -rf ${pkgs.strongswan}/etc/swanctl/private
+      rm -rf ${pkgs.strongswan}/etc/swanctl/rsa
+      rm -rf ${pkgs.strongswan}/etc/swanctl/ecdsa
+      rm -rf ${pkgs.strongswan}/etc/swanctl/bliss
+      rm -rf ${pkgs.strongswan}/etc/swanctl/pkcs8
+      rm -rf ${pkgs.strongswan}/etc/swanctl/pkcs12
+      ln -s /etc/strongswan.conf ${pkgs.strongswan}/etc/strongswan.conf
+      ln -s /etc/strongswan.d ${pkgs.strongswan}/etc/strongswan.d
+      ln -s /etc/swanctl/swanctl.conf ${pkgs.strongswan}/etc/swanctl/swanctl.conf
+      ln -s /etc/swanctl/x509 ${pkgs.strongswan}/etc/swanctl/x509
+      ln -s /etc/swanctl/x509ca ${pkgs.strongswan}/etc/swanctl/x509ca
+      ln -s /etc/swanctl/x509ocsp ${pkgs.strongswan}/etc/swanctl/x509ocsp
+      ln -s /etc/swanctl/x509aa ${pkgs.strongswan}/etc/swanctl/x509aa
+      ln -s /etc/swanctl/x509ac ${pkgs.strongswan}/etc/swanctl/x509ac
+      ln -s /etc/swanctl/x509crl ${pkgs.strongswan}/etc/swanctl/x509crl
+      ln -s /etc/swanctl/pubkey ${pkgs.strongswan}/etc/swanctl/pubkey
+      ln -s /etc/swanctl/private ${pkgs.strongswan}/etc/swanctl/private
+      ln -s /etc/swanctl/rsa ${pkgs.strongswan}/etc/swanctl/rsa
+      ln -s /etc/swanctl/ecdsa ${pkgs.strongswan}/etc/swanctl/ecdsa
+      ln -s /etc/swanctl/bliss ${pkgs.strongswan}/etc/swanctl/bliss
+      ln -s /etc/swanctl/pkcs8 ${pkgs.strongswan}/etc/swanctl/pkcs8
+      ln -s /etc/swanctl/pkcs12 ${pkgs.strongswan}/etc/swanctl/pkcs12
+     '';
+
+    launchd.daemons.strongswan-swanctl = {
+      script = "
+        ${pkgs.strongswan}/sbin/ipsec start --nofork && ${cfg.package}/sbin/swanctl --load-all --noprompt
+      ";
+      environment = {
+        SWANCTL_DIR = "/etc/swanctl";
+      };
+      serviceConfig = {
+        RunAtLoad = true;
+        KeepAlive.NetworkState = true;
+        StandardErrorPath = "${config.launchd.daemons.strongswan-swanctl.environment.SWANCTL_DIR}/buildkite-agent.log";
+        StandardOutPath = "${config.launchd.daemons.strongswan-swanctl.environment.SWANCTL_DIR}/buildkite-agent.log";
+      };
+    };
+  };
```

```suggestion
  }) ];
```
With the previous code suggestion